src/vmm_unix.ml

```diff
@@ -374,23 +377,26 @@ let mb_of_bytes size =
 
 let find_block_devices () =
   let blockdir = Fpath.(!dbdir / block_sub) in
-  Bos.OS.Dir.contents ~rel:true blockdir >>= fun files ->
-  List.fold_left (fun acc file ->
-      acc >>= fun acc ->
-      let path = Fpath.append blockdir file in
-      Bos.OS.File.exists path >>= function
-      | false ->
-        Logs.warn (fun m -> m "file %a doesn't exist, but was listed" Fpath.pp path) ;
-        Ok acc
-      | true ->
-        Bos.OS.Path.stat path >>= fun stats ->
-        match mb_of_bytes stats.Unix.st_size, Name.of_string (Fpath.to_string file) with
-        | Error (`Msg msg), _ ->
-          Logs.warn (fun m -> m "file %a size error: %s" Fpath.pp path msg) ;
-          Ok acc
-        | _, Error (`Msg msg) ->
-          Logs.warn (fun m -> m "file %a name error: %s" Fpath.pp path msg) ;
+  Bos.OS.Path.exists blockdir >>= function
+  | false -> Bos.OS.Dir.create blockdir >>| fun _ -> []
```

Comment:
We probably want to pass `~mode:0o700` or something:
`[create ~path ~mode dir] creates, if needed, the directory [dir] with file permission [mode] (defaults [0o755] readable and traversable by everyone, writeable by the user).`

Reply:
albatrossd (executing this function) is running as root (for now, in order to create virtual machines). I'm not entirely sure about the permissions that should be in place for block devices (and unikernel image files in /var/run/albatross)... there's no need for others to read/write/execute any of that...

Reply:
I would expect the block devices to not be readable by any other users, and probably the same for the image files.

Should we make a separate issue with permissions review and ignore it for now?

Reply:
maybe we leave this PR open until all the permission and exceptions are dealt with, shouldn't take too long, and better be done with it.

Reply:
this is now solved - installation scripts create (with permission 0700), the create_block creates with 0700 (files with 0600).